src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoLineWrapCheck.java

```diff
@@ -117,7 +118,7 @@
 
     @Override
     public void visitToken(DetailAST ast) {
-        if (ast.getLineNo() != ast.getLastChild().getLineNo()) {
+        if (!TokenUtil.areOnSameLine(ast, ast.getLastChild())) {
```

Comment:
Does it make sense to make method `areOnDifferentLines` ? in addition ...
(If yes, coul)
 
Reason:
`x != y` is reading easily  but `!TokenUtil.someSomeSomeOnSome` is not as `!` is far from actual context.

Reply:
Maintaining two methods (even single-line ones) will be twice as hard.

Reply:
It is not so common to add "negation" methods like `notBlaBla` in java, but in scala people do that, as far as I know.
I like this approach (use negate methods), as for me method like `notOnSameLine` would be more readable than negation.

Reply:
I didn't propose negation, I used word with opposite meaning.

Usage of "not" is not very good if there is a way to use opposite word, it is ok especially if there is sibling method with positive name. In our case we do have such sibling method, so we will not force to use double negation.

I do not think there will be any maintenance costs for extra method, especially if we place them close to each other.

We spoke a lot about methods that starts with "no", I think we are ready to make Check on this in sevntu project, in config user need to be able to provide map of method sibling to skip violation on method that starts with negation.
If team agree, I can start issue, to keep this idea written.

Reply:
@rnveach , @pbludov , what are your thoughts on this

Reply:
We will need to use this rule in all similar cases. This can be difficult.

Reply:
@rnveach , @pbludov , 

Does it make sense to make method areOnDifferentLines ?

Reply:
> Does it make sense to make method areOnDifferentLines ?

I vote against adding the method `areOnDifferentLines`.

Reply:
Truthfully, I don't see the point of this type of method. Its just a single condition so it seems like more work than to just compare the values, especially when the condition looks as simple as the possible new method.

Reply:
Ok, I will come back to this in next PR, as I do not like negation a lot

Reply:
https://github.com/sevntu-checkstyle/sevntu.checkstyle/issues/802